Header header

uint32 timesFired
uint32 state
